src/main.js

```diff
@@ -55,8 +55,17 @@ if (window.location.search.indexOf("testEnvironment") > -1) {
      * navigator.language in ExtensionLoader (when making require contexts for each
      * extension).
      */
+    var locale;
+    var regex = /locale=([^&]+)&?/.exec(window.location.search);
+    if(window.location.search && regex !== null) {
```

Comment:
In fact, you could put that second line into the `require.conf()` call directly:

```js
var urlLocale = window.location.search && /locale=([^&]+)&?/.exec(window.location.search);

require.config({
  locale: !!urlLocale ? urlLocale[1] : (typeof (brackets) !== "undefined" ? brackets.app.language : navigator.language);
});
```